packages/client/src/navigation/index.ts

```diff
@@ -183,6 +183,13 @@ export function goToHomeTab(tabId: IWORKQUEUE_TABS, selectorId = '') {
   return push(formatUrl(REGISTRAR_HOME_TAB, { tabId, selectorId }))
 }
 
+export function goToHomeFromCorrectionTab(
```

Comment:
Can you please rename the function to something like `goToHomeTabReplace`? Because if it is called in any other component other than Correction it will appear misleading

Reply:
alright...

Reply:
alright..